build.gradle.kts

```
@@ -31,6 +32,8 @@ allprojects {
 }
 
 dependencies {
+    implementation(libs.jacksonModuleKotlin)
+    implementation(libs.jacksonDataformatYaml)
```

nit: alphabetical order and separation of dependency types

```suggestion
    implementation(libs.jacksonDataformatYaml)
    implementation(libs.jacksonModuleKotlin)

```